src/components/SideBar.jsx

```diff
@@ -20,6 +21,11 @@ const Sidebar = ({ children }) => {
   const [activeUserRole, setActiveUserRole] = useState(null);
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const {
+    showConfirmation,
+    handleLogoutIconClick,
+    renderLogoutConfirmation,
+  } = setLogout();
   useEffect(() => {
     if (location) {
```

Comment:
add this logout functionality on navbar for dashboard, because this sidebar will be removed